### ISSUE TITLE
Ram usage with XfreeRDP

### ISSUE THREAD

**Issue by wkmdd** [FreeRDP/FreeRDP#7071]
Hi,

we're using FreeRDP version 3.0.0-dev on a Puppy-Linux Client for the connection to our Windows TS 2012R2.
 
via:
/opt/freerdp-nightly/bin/xfreerdp /v:XXXX /f /d:XXXX /u:XXXX /p:XXXX  /monitors:0,1 +clipboard /microphone:sys:alsa,format:1,rate:44100 /sound /auto-reconnect /auto-reconnect-max-retries:10 /kbd:0x00000407 /dynamic-resolution /gfx:AVC420 
 
This works fine in the beginning, but after a while the FreeRDP process starts using increasingly more RAM, which due to hardware limitations results in a freeze.
 
At the start of the connection the process uses about 100 MBit RAM and with every input (i. e. opening/closing a window) it grows. 
This goes on until it reaches about 500MBit, after that the Process starts clearing every extra RAM used.
So in conclusion every FreeRDP Process uses about 500MBit RAM at its max.

Is there a way to change this behavior, for example changing the RAM limit?

**Comment by akallabeth** [comment]
not use nightly builds but build your own?
we provide nightlies so that they are most useful for finding bugs and get proper error reports.
that includes `ASAN` and other memory intensive compiler options.
